webpack.config.js

```diff
@@ -11,6 +11,27 @@ module.exports = {
         __dirname: false,
         __filename: false,
     },
+    module: {
+        rules: [
+            {
+                exclude: /(node_modules)/,
+                use: {
+                    loader: 'babel-loader',
+                    options: {
+                        presets: ['@babel/preset-env'],
+                        plugins: [
+                            [
+                                "@babel/plugin-transform-modules-commonjs", {
+                                    "allowTopLevelThis": true
```

Comment:
If we remove the `allowTopLevelThis: true` will the tests still pass?

Reply:
Unsure, let's check.

Reply:
Looks like it passed :+1: